### ISSUE TITLE
ml-dtypes does not build from sources on rockylinux 8 with gcc-c++ 8.5

### ISSUE THREAD

**Issue by thebaptiste** [jax-ml/ml_dtypes#69]
It fails because of `#include <version>` in `float8.h` (file `version` is not found)
I think it's not compliant with `C++17` but with `C++20`
May be you could do something like this : 
```
#ifdef __has_include
# if __has_include(<version>)
#   include <version>
# endif
#endif
```

**Comment by thebaptiste** [comment]
I tried a pull request : #70

**Comment by hawkinsp** [comment]
I'm guessing this is fixed, since we merged #70. Thanks for the PR!
